tournament/src/tournament/authenticate_request.py

```diff
@@ -36,12 +37,14 @@ def decode_jwt(encoded_jwt, errors):
 def get_user(user_id, encoded_jwt, errors):
     headers = {'Authorization': encoded_jwt}
     try:
-        response = requests.get(f'http://user-management-nginx/user/{user_id}/', headers=headers)
+        response = requests.get(f'http://127.0.0.1:8001/user/{user_id}/', headers=headers)
```

Comment:
Define the address in settings.

Reply:
794ccceacaf81fb73290cac30f20b983921f4a69